.github/workflows/azure-static-web-apps-white-bush-06149C11E.yml

```diff
@@ -0,0 +1,52 @@
+name: Azure Static Web Apps CI/CD
+
+on:
+  workflow_dispatch:
+  push:
+    paths: 
+      - wordle-web/**
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  build_and_deploy_job:
+    runs-on: ubuntu-latest
+    name: Build and Deploy Job
+    steps:
+      - name: Checkout Code 
+        uses: actions/checkout@v3
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - name: Install Packages
+        run: npm ci
+        working-directory: ./wordle-web
+      - name: Lint Code
+        working-directory: ./wordle-web
+        run: npm run lint
+      - name: Prettier Check
+        working-directory: ./wordle-web
+        run: npm run format:check
+      - name: Build Vue App 
+        working-directory: ./wordle-web
+        run: npm run build
+      - name: Run Vitest Unit Tests
+        run: npm run test:unit
+        working-directory: ./wordle-web
+     
+      - name: Deploy
+        if: (github.event_name == 'push' || github.event_name == 'workflow_dispatch')
+        id: builddeploy
+        uses: Azure/static-web-apps-deploy@v1
+        with:
+          azure_static_web_apps_api_token: ${{ secrets.AZURE_STATIC_WEB_APPS_API_TOKEN_VICTORIOUS_POND_0D6944210 }}

```

Comment:
Remember to change this so it does not have your token name. So, it should just be secrets.AZURE_STATIC_WEB_APPS_API_TOKEN. You can do this by: 
1. Go to Azure
2. Copy your secret token
3. Go to GitHub 
4. Go to your repository and branch that you pushed too
5. Click Secrets and Variables
6. Click Actions
7. Click New Repository Secret
8. Paste your token from Azure and name it AZURE_STATIC_WEB_APPS_API_TOKEN

Reply:
hey thanks for the detailed response here - getting this thing deployed is first on my todo list!